src/appengine/handlers/upload_testcase.py

```diff
@@ -322,12 +322,10 @@ def do_post(self):
 
     fuzzer_name = request.get('fuzzer')
     job_type_lowercase = job_type.lower()
-    if 'libfuzzer' in job_type_lowercase:
-      fuzzer_name = 'libFuzzer'
-    elif 'afl' in job_type_lowercase:
-      fuzzer_name = 'afl'
-    elif 'honggfuzz' in job_type_lowercase:
-      fuzzer_name = 'honggfuzz'
+
+    for engine in fuzzing.ENGINES:
```

Comment:
Kind of nervous someone might one day have multiple engine names in the job name.